Program.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+import java.util.regex.Pattern;
+public class Program {
+ 
+    public static void main(String[] args) {
+       Scanner in = new Scanner (System.in); 
+       System.out.print("Input a sentence: ");
+       String sentence = in.nextLine();
+       in.close();
+       StringBuffer strBuf = new StringBuffer (sentence);
+       
+       RevOrder (strBuf);
+       System.out.println("==========");
+       Words (sentence);
+       System.out.println("==========");
+       ReplaceSent (sentence);
+       System.out.println("==========");
+       GettingSent (sentence);
+       System.out.println("==========");
+       
+      
+    }
+    
+    //Reverse Order
+    static void RevOrder (StringBuffer Order) {
```

Comment:
it will be more "object-oriented" to pass to RevOrder method a string, a to create StringBuffer in the method body as only this method is using it.